macros/src/actix/mod.rs

```diff
@@ -169,6 +173,36 @@ pub fn emit_v2_definition(attrs: TokenStream, input: TokenStream) -> TokenStream
     gen.into()
 }
 
+fn get_field_type(field: &Field, mut is_required: Option<bool>) {
+    match field.ty {
+        Type::Path(ref p) => {
+            let ty = p
+                .path
+                .segments
+                .last()
+                .expect("expected type for struct field");
+
+            if p.path.segments.len() == 1 && &ty.ident == "Option" {
+                is_required = Some(false);
+            }
+
+            address_type_for_fn_call(&field.ty)
+        }
+        Type::Reference(_) => address_type_for_fn_call(&field.ty),
+        _ => address_type_for_fn_call(&field.ty),
```

Comment:
not fully sure about this 🤔

Reply:
This should be `None`, since we don't care other kinds of types just yet. It looks like you've removed the warning we used to emit.